DirectProgramming/DPC++/N-BodyMethods/Nbody/sample.json

```diff
@@ -0,0 +1,29 @@
+{
+  "guid": "D5BFD7D5-737E-4CC9-84EE-FE94339B3DCA"
```

Comment:
Incorrect Syntax, comma is missing, sample will not pass CI. Please run jsonlint on this file before submitting merge request.